### PR TITLE
re-create Homepage

### PR DESCRIPTION
**Pull request by agrawalshanu708**
Re-create home page design

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -8,6 +8,7 @@
 .App {
 display: flex;
 flex-direction: column;
+text-align: center;
 /* background-color: var(--text-white-color) */
```
Comment (ckahub) [inline src/App.css:12]: remove coment code

File: src/components/Navbar/Navbar.css [hunk 2]
```diff
@@ -5,27 +5,28 @@
 
 .navbar{
     display:flex;
-    box-shadow: 0 1rem 1rem rgba(0,0,0,0.12), 0 1rem 1rem rgba(0,0,0,0.24);
+    /* box-shadow: 0 1rem 1rem rgba(0,0,0,0.12), 0 1rem 1rem rgba(0,0,0,0.24); */
```
Comment (ckahub) [inline src/components/Navbar/Navbar.css:8]: remove coment part

File: src/pages/Home/homepage.css [hunk 3]
```diff
@@ -56,4 +53,102 @@
         height: 100%;
         width: 100%;
     }
+}
+ /* ------------------------------------------------- */
+ .banner-boxy{
+     /* border: 1px solid black; */
```
Comment (ckahub) [inline src/pages/Home/homepage.css:59]: same

**Review by harshitpaliwal95 — APPROVED** [review]
Looks good you can merge

File: src/components/Navbar/Navbar.css [hunk 4]
```diff
@@ -5,27 +5,28 @@
 
 .navbar{
```
Comment (Chiragmodi01) [inline src/components/Navbar/Navbar.css:6]: Nav pills are overlapping screen's text or image, that's not a good user experience to me. You can give the background inset shadow or light color

File: src/components/Navbar/Navbar.jsx [hunk 5]
```diff
@@ -12,18 +12,19 @@ const  Navbar = () => {
...
-               placeholder="Search for Products,Brands.."
-               />
-               <span class="material-icons search-icon">search</span>       
+                <p>Home</p>
+                <p>Products</p>
+                <p>Wishlist</p>
+                      
                </div>
                <div class="nav-right-head">
+               <span class="material-icons search-icon">search</span>
```
Comment (Chiragmodi01) [inline src/components/Navbar/Navbar.jsx:21]: I see below you're using react-icons are some other library here. Would recommend using one only, that'll unnecessarily increase the load on your website

File: src/pages/Home/homepage.css [hunk 6]
```diff
@@ -56,4 +53,102 @@
         height: 100%;
         width: 100%;
     }
+}
+ /* ------------------------------------------------- */
+ .banner-boxy{
```
Comment (Chiragmodi01) [inline src/pages/Home/homepage.css:58]: this class name could be improvise 'boxy' ><

File: src/pages/Home/homepage.css [hunk 7]
```diff
@@ -56,4 +53,102 @@
...
+    /* border: 1px solid black; */
+    display: flex;
+    flex-direction: column;
+    gap: 4rem;
+    align-items: center;
+    justify-content: center;
+}
+.banner-heading{
+    margin-top: 5rem;
+    font-family: 'Lato', sans-serif;
```
Comment (Chiragmodi01) [inline src/pages/Home/homepage.css:72]: You've used this font-family many times in your code, you can make a utility class for this for just use variable.

File: src/pages/Home/homepage.jsx [hunk 8]
```diff
@@ -1,81 +1,106 @@
...
 
-            <div className="banner-box col-12">
-            <Link to="/products" > <img src={banner1} alt=""/></Link>
+          <div className="cards-outer-box">
+            <div className="headline">Trending Items</div>
+            <div className="cards-box">
+            
+            <div className="card">
+              <div className="card-img-box">
+           <img src= "https://assets.myntassets.com/f_webp,dpr_1.5,q_60,w_210,c_limit,fl_progressive/assets/images/9460049/2019/10/5/c47398e2-72a5-45a4-bcb5-dd58adee9b161570274183766-Roadster-Men-Black-Multi-Function-Analogue-and-Digital-Watch-1.jpg" alt="" />
```
Comment (Chiragmodi01) [inline src/pages/Home/homepage.jsx:46]: This huge image source looks messy to me in between the main file's code. you can store them separately.

File: src/pages/Home/homepage.jsx [hunk 9]
```diff
@@ -1,81 +1,106 @@
...
+           <div className="card">
+              <div className="card-img-box">
+           <img src= "https://assets.myntassets.com/f_webp,dpr_1.5,q_60,w_210,c_limit,fl_progressive/assets/images/17161252/2022/2/15/535b9721-0249-4f06-8054-7adfdc6462131644916584800EmporioArmaniMenBlackSkeletonDialBlackStainlessSteelStrapsAn1.jpg" alt="" />
+           </div>
+            <p>Titan</p>
+           <button className="home-btn">Add To Cart</button>
+           </div>
+           <div className="card">
+              <div className="card-img-box">
+           <img src= "https://assets.myntassets.com/f_webp,dpr_1.5,q_60,w_210,c_limit,fl_progressive/assets/images/17161230/2022/2/15/5e452a7f-fb07-4080-acec-5ae580d4478a1644916949644EmporioArmaniMenBlueDialSilverTonedStainlessSteelBraceletSty1.jpg" alt="" />
```
Comment (Chiragmodi01) [inline src/pages/Home/homepage.jsx:60]: alt missing for almost all of the images

File: src/pages/Home/homepage.jsx [hunk 10]
```diff
@@ -1,81 +1,106 @@
...
+         {/* ---------------------------------- */}
+         <h2>Brands</h2>
      <div className="brand-container">
          <img src= {brand1} alt="" />
          <img src={brand2} alt="" />
          <img src={brand3} alt="" />
          <img src={brand4} alt="" />
          <img src={brand5} alt="" />
      </div>
+     {/* -------------------------- */}
```
Comment (Chiragmodi01) [inline src/pages/Home/homepage.jsx:97]: I don't this it's necessary to used this commented separation in your jsx file, usual practice is to use it in css.

**Review by Chiragmodi01 — COMMENTED** [review]
Home page's UI looks good to me. Also commented code can be removed.
Just added few sanity comments.

**Pull request merged**